app/syncables.go

```diff
@@ -36,6 +36,15 @@ func (a *App) createDefaultChannelMemberships(since int64, channelID *string) er
 		if tmem == nil {
 			_, err = a.AddTeamMember(channel.TeamId, userChannel.UserID)
 			if err != nil {
+				if err.Id == "api.team.join_user_to_team.allowed_domains.app_error" {
+					a.Log().Info("Not adding user to channel because the domain associated with the account is not permitted in the given team",
```

Comment:
@thefactremains What do you think of this message? Any changes you want to it?

Reply:
I prefer this:
**User not added to channel - the domain associated with the user is not in the list of allowed team domains**

Reply:
Updated @thefactremains